src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/form.html.twig

```diff
@@ -442,7 +439,7 @@
             {% endif %}
 
             <span {% if has_error or data.has_children_error|default(false) %}class="has-error"{% endif %}>
-                {{ name|default('(no name)') }} {% if data.type_class is defined %}[<abbr title="{{ data.type_class }}">{{ data.type_class|split('\\')|last }}</abbr>]{% endif %}
+                {{ name|default('(no name)') }} [{{ profiler_dump(data.type_class) }}]
```

Comment:
Indeed, I missed a few CSS here. I did something similar for log contexts, they are inline, while still in <pre>, we should do the same here also.